fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
Okay, well, if it's blocking OSGi, that's a good reason to remove it in favor of docs. Why on Earth is MODE loading Cassandra drivers?